app/helpers/application_helper.rb

```diff
@@ -21,4 +21,12 @@ def branding_image
     asset_path = SettingGetter.new(setting_name: 'BrandingImage', provider: current_provider).call
     asset_url(asset_path)
   end
+
+  def page_title
+    match = request&.url&.match('\/rooms\/(\w{3}-\w{3}-\w{3}-\w{3})')
+    return 'BigBlueButton' if match.blank?
+
+    room_name = Room.find_by(friendly_id: match[1])&.name
+    room_name || 'BigBlueButton'
```

Comment:
Since opengraph won't run any JS on the page, we need to dynamically pull the room name on the server side for it to show in open graph